sal_interfaces/MTCamera/MTCamera_Telemetry.xml

```diff
@@ -1,948 +1,3407 @@
+            <Units>Amps</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>compPower</EFDB_Name>
+            <Description>Compressor Power</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>VA</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>compVfdVolt</EFDB_Name>
+            <Description>Compressor VFD Voltage</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Volt</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>compVfdCurr</EFDB_Name>
+            <Description>Compressor VFD Current</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Amps</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>compVfdFreq</EFDB_Name>
+            <Description>Compressor VFD Frequency</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Hz</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>hGBValvePosn</EFDB_Name>
+            <Description>HGB Valve Position</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>%</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>coolValvePosn</EFDB_Name>
+            <Description>Coolant Valve Position</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>%</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>coolFlowRate</EFDB_Name>
+            <Description>Coolant Flow Rate</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>l/min</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>waterInTmp</EFDB_Name>
+            <Description>Water Inlet Temperature</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>waterOutTmp</EFDB_Name>
+            <Description>Water Outlet Temperature</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>cabinetTmp</EFDB_Name>
+            <Description>Cabinet Temperature</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>ambientTmp</EFDB_Name>
+            <Description>Ambient Temperature</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>Celsius</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>fan1Speed</EFDB_Name>
+            <Description>Fan 1 Speed (RPM)</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units/>
```

Comment:
Units seem to be missing here and the next two entries. Is it because RPM isn't supported by default?

Reply:
Fixed. I think we previously decided RPM was not a unit and should by included in the title, and that unit should be unitless (which it will be after the next push).